api/src/openapi/schemas/pagination.ts

```diff
@@ -0,0 +1,79 @@
+import { SchemaObject } from 'ajv';
+
+/**
+ * API schema used to assert pagination query paramaters
+ * for paginated data requests.
+ */
+export const paginationRequestQueryParamSchema: any[] = [
+  {
+    in: 'query',
+    name: 'page',
+    required: false,
+    schema: {
+      type: 'integer',
+      minimum: 1,
+      description: 'The current page number being fetched'
+    }
+  },
+  {
+    in: 'query',
+    name: 'limit',
+    required: false,
+    schema: {
+      type: 'integer',
+      minimum: 1,
+      maximum: 100,
+      description: 'The number of records per page'
+    }
+  },
+  {
+    in: 'query',
+    name: 'sort',
+    required: false,
+    description: 'The column being sorted on'
+  },
+  {
+    in: 'query',
+    name: 'order',
+    required: false,
+    description: 'The order of the sort, i.e. asc or desc'
+  }
+];
+
+/**
+ * API schema to assert pagination information for paginated data
+ * responses.
+ */
+export const paginationResponseSchema: SchemaObject = {
+  type: 'object',
+  required: ['total', 'current_page', 'last_page'],
+  properties: {
+    total: {
+      type: 'integer',
+      description: 'The total number of observation records belonging to the survey'
```

Comment:
nit: is this description meant to be generic?

Reply:
Good catch, I've changed it.